clusterloader2/testing/density/config.yaml

```diff
@@ -26,14 +26,10 @@ tuningSets:
     qps: 5
 steps:
 - measurements:
-  - Identifier: APIResponsiveness
-    Method: APIResponsiveness
+  - Identifier: TestMetrics
```

Comment:
APIResponsiveness should be defined explicitly - (at least for now).

Medium term, we may create a bundle for "slo validation" or sth like that.

Reply:
Done.